zerver/lib/events.py

```diff
@@ -208,6 +208,7 @@ def fetch_initial_state_data(
         state["realm_bot_domain"] = realm.get_bot_domain()
         state["realm_uri"] = realm.uri
         state["realm_available_video_chat_providers"] = realm.VIDEO_CHAT_PROVIDERS
+        state["realm_giphy_rating_options"] = realm.GIPHY_RATING_OPTIONS
```

Comment:
I think probably `giphy_rating_options` is the right name for the field, similar to `password_min_guesses` below.

Reply:
Done.